src/main/java/objectsBasicsMandatoryPracticalTasks/Dogs/Dog.java

```diff
@@ -0,0 +1,44 @@
+package objectsBasicsMandatoryPracticalTasks.Dogs;
+
+class Dog {
+    private String name, age, size;
+
+    Dog(String name, String age, String size) {
+        this.name = name;
+        this.age = age;
+        this.size = size;
+        this.printDog();
+    }
+
+    static String getRandomElement(String[] arrName) {
```

Comment:
From best practices it would be better to have separate "utility" class like RandomElement with this method.
It doesn't have anything in common with `Dog` class, so it could be applied to any String array